src/controllers/busStop.js

```diff
@@ -0,0 +1,98 @@
+import  model  from '../database/models'
+import succesRes from '../helpers/successHandler'
+import errorRes from '../helpers/errorHandler'
+import cryptoRandomString from 'crypto-random-string'
+
+import { Router } from 'express';
+const router = Router();
+
+const { busStop } = model;
+
+
+const createBusStop = async (req,res) => {
+    try{
+        let Stop_id = cryptoRandomString({length: 5, type: 'numeric'})
+        const searchBusStop = await busStop.findAll( { where : req.body } );
+        const searchBusStopId = await busStop.findAll( { where : { busStopId : Stop_id } } )
+        if(searchBusStop.length === 0 && searchBusStopId.length === 0){
+            const Stop = await busStop.create({
+                coordinates : req.body.coordinates,
+                sector : req.body.sector,
+                cell : req.body.cell,
+                busStopId: Stop_id,
+            });
+            return succesRes(res,201,"Bus stop created successfully" , Stop);
+        } else{
+            return errorRes(res,404,"Bus stop already exists");
+        }
+    } catch (error) {
+        return errorRes(res,500,"Internal server error : " + error.message);
+    }
+}
+
+const getBusStops =  async (req,res) => {
+    try{
+        const Stop = await busStop.findAll();
```

Comment:
could you change stop to busStops please? We should use the meaningful variables declaration

Reply:
Thanks ray, Worked on that!